homeassistant/components/denonavr/media_player.py

```diff
@@ -150,6 +152,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
             _LOGGER.info("Denon receiver at host %s initialized", host)
 
     # Add all freshly discovered receivers
+    hass.data[DOMAIN]["receivers"] = receivers
```

Comment:
Don't store the entities in a shared container. It's ok to store references to the entities eg the entity_ids.